freqtrade/freqai/data_kitchen.py

```diff
@@ -87,9 +87,13 @@ def __init__(
             )
 
         if self.live:
-            db_url = self.config.get('db_url', 'sqlite://')
-            self.database_path = '' if db_url == 'sqlite://' else str(db_url).split('///')[1]
-            self.trade_database_df: DataFrame = pd.DataFrame()
+            db_url = self.config.get('db_url', 'None')
+            self.database_path = Path(db_url)
+            self.database_name = self.database_path.parts[-1]
+        else:
+            self.database_path = Path('None')
```

Comment:
a better approach (especially considering the check you need to do) would be to assign  the following  - but before the `if`.
That'd enable mypy to tell you where you need to check if the database_path is not set properly.

```
        self.database_path: Optional[Path] = None
```